salt/utils/event.py

```diff
@@ -177,14 +177,15 @@ class SaltEvent(object):
     '''
     def __init__(
             self, node, sock_dir=None,
-            opts=None, listen=True, io_loop=None):
+            opts=None, listen=True, io_loop=None, keep_loop=False):
         '''
         :param IOLoop io_loop: Pass in an io_loop if you want asynchronous
                                operation for obtaining events. Eg use of
                                set_event_handler() API. Otherwise, operation
                                will be synchronous.
         '''
```

Comment:
I'm totally going to forget why we did this one day without a docstring in here that outlines the intention of this flag. ;]